functions/validationResult.R

```diff
@@ -8,21 +8,21 @@ validationResult <- function(anno.res, template, manifest) {
   error_type <- NULL
   highlight_values <- list()
 
-  # in case there are some errors we haven't captured
-  if (is.null(anno.res)) {
+  # if no uploaded manifest or empty manifest
+  if (is.null(manifest) || nrow(manifest) == 0) {
     return(list(
       result = "invalid",
-      error_type = "Out of Date",
-      error_msg = "Something went wrong while validating your manifest. Please contact administrator."
+      error_type = "Empty File",
+      error_msg = "Please <b>upload</b> a filled template !"
     ))
   }
 
-  # if no uploaded manifest or empty manifest
-  if (is.null(manifest) || nrow(manifest) == 0) {
+  # in case there are some errors we haven't captured
+  if (is.null(anno.res)) {
     return(list(
       result = "invalid",
-      error_type = "Empty File",
-      error_msg = "Please <b>upload</b> a filled template !"
+      error_type = "Out of Date",
```

Comment:
How flexible is the name of this error_type? Since this is a hot fix the "out of date" type is fine, but we probably want something more generic in general.

Reply:
This error is occurred when the validationModelManifest failed and used to prevent the app from crashing. It was not always clear what the errors were coming from. It could be the the data type they choose from dropdown is invalid for validation anymore, or new incompatible errors with updated schematic. 

The `error_type` is display on the waiter screen after the validation completes, before they can see the actual `error_msg`. I agree to improve the error type for this scenario to a more generic language in the future.